bitnami/redis/templates/configmap.yaml

```diff
@@ -8,7 +8,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ printf "%s-configuration" (include "common.names.fullname" .) }}
-  namespace: {{ .Release.Namespace | quote }}
+  namespace: {{ include "common.names.namespace" . }}
```

Comment:
Please keep the quote function. Some users have "numeric" namespaces.
```suggestion
  namespace: {{ include "common.names.namespace" . | quote }}
```

Also keep it mind in other yaml templates.

Reply:
I believe we can solve this globally by changing this in the common chart:

![image](https://github.com/bitnami/charts/assets/38658722/fa059126-b6da-4436-b808-16691ec92b8a)

But this would require a separate PR I assume (Bump chart and reference the new version here)

Reply:
This way, namespace will always be "<VALUE>" no matter the type

Reply:
@jgmartinez This is precisely the point. 
The namespace is a DNS label and must comply with RFC 1123.
Ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names

Reply:
check #21413